fastly/vcl/polyfill-service.vcl

```diff
@@ -81,6 +81,39 @@ sub vcl_recv {
 		call breadcrumb_recv;
 	}
 
+
+	declare local var.compute-at-edge-active BOOL;
+	if (std.atoi(table.lookup(compute_at_edge_config, "active", "0")) == 1) {
+		set var.compute-at-edge-active = true;
+	} else {
+		set var.compute-at-edge-active = false;
+	}
+	if (var.compute-at-edge-active) {
+		# if the querystring parameter `use-compute-at-edge-backend` is set to yes`
+		# then use the c@e backend.
+		# else if the `use-compute-at-edge-backend` is not set to `no` then make
+		# 1 in 1000 requests which come directly from a client use the c@e backend
```

Comment:
```suggestion
		# The querystring parameter is used within our tests, so that we can confirm 
		# the compute-at-edge backend is working as we expect it to.
```